Pcap++/src/PcapLiveDeviceList.cpp

```diff
@@ -231,7 +229,7 @@ PcapLiveDevice* PcapLiveDeviceList::getPcapLiveDeviceByIp(IPv4Address ipAddr) co
 				continue;
 			}
 
-			if (currAddr->s_addr == ipAddr.toInAddr()->s_addr)
+			if (memcmp(currAddr, ipAddr.toBytes(), sizeof(*currAddr)) == 0)

```

Comment:
why using `memcmp` here instead of comparing `currAddr->s_addr` with `ipAddr.toUInt()`?

Reply:
For IPv6 address the following check is used:
```c++
if (memcmp(currAddr, addrAsArr, sizeof(struct in6_addr)) == 0)
```

I just made the check for IPv4 in the same way. The possible replacement is:
```c++
if (currAddr->s_addr == ipAddr.toUInt())
```

Reply:
Should I remove `memcmp`?

Reply:
`memcmp` was removed